src/renderer/components/SyncView/index.js

```diff
@@ -138,8 +138,8 @@ function usePeers (listen) {
           updatedServerPeers.forEach(peer => {
             if (!peer.state) return
             if (
-              peer.state.topic === 'replication-error' ||
-              peer.state.topic === 'replication-complete'
+              (peer.state.topic === 'replication-error' ||
+              peer.state.topic === 'replication-complete') && !peer.connected
```

Comment:
If a sync request is sent very quickly after a syncronization is complete, this code was removing that sync request before it even started again. The sync request should only be deleted once it is an error or completion AND the peer has disconnected, which means that the entire lifecycle of the sync has completed.

Reply:
This likely will not be an issue with mapeo-core v9, however